knowledge-base/how-tos/0006-set-up-identity-service.md

```diff
@@ -146,3 +146,34 @@ The payload of the decoded IAS (JWT) looks like below, where `user_uuid` (IdP `u
   "email": "xxx"
 }
 ```
+
+#### Verify IAS token via HTTP request
+
+You can use the client id/secret for getting the IAS token via the token endpoint (`/oauth2/token`), after creating the service key in step 2.
+We assume the domain of your default IdP looks like `https://xx.xx.ondemand.com/`.
```

Comment:
Is this really the IdP domain or the XSUAA? For all other services like destination you call the XSUAA.

Reply:
This the default IdP, as so far I only used the `identity service` without any XSUAA service binding. The default IdP is bound to the `identity service` if you create an `identity service key`.